tests/integration/test_minidump.py

```diff
@@ -70,8 +70,8 @@ def test_minidump_attachments(mini_sentry, relay):
     mini_sentry.project_configs[project_id] = mini_sentry.full_project_config()
 
     event = {"event_id": "2dd132e467174db48dbaddabd3cbed57", "user": {"id": "123"}}
-    breadcrumbs1 = [{"timestamp": 1461185755, "message": "A",}]
-    breadcrumbs2 = [{"timestamp": 1461185750, "message": "B",}]
+    breadcrumbs1 = {"timestamp": 1461185755, "message": "A",}
```

Comment:
This change ensures that we don't serialize an enclosing array, but instead serialize the breadcrumb directly. We could also serialize multiple of those in a loop, but this is done in the rust tests already.